### ISSUE TITLE
lets test email notifications 

### ISSUE THREAD

**Issue by tankf33der** [TrustInSoft-CI/TrustInSoft-CI#14]
@guillaumemillot 
I've tried to trigger to get notification via "All failing verifications" and "All verifications" in existing projects and no mail in 5mins.
Ask infrastructure slackers and let me try again :)

**Comment by guillaumemillot** [comment]
Thanks Mike for testing all our new features 👍 
The issue is related to TIS CI failing, in some cases, to record the user's email provided by GitHub upon sign-up.
A fix is under way, I'll keep you posted!

**Comment by guillaumemillot** [comment]
A fix has been pushed. Do you mind doing a quick test? Thanks!

**Comment by tankf33der** [comment]
@guillaumemillot 
Tested in all combinations of tasks and settings, works as expected except one thing:
Email settings does not respect cancelled tasks, message should not send only with value "None".

**Comment by guillaumemillot** [comment]
Do you mind forwarding me the email for a cancelled verification?

**Comment by tankf33der** [comment]
I was not fully clear in prev. message - i did not received messages for cancalled jobs, but i should, so nothing to forward.

**Comment by guillaumemillot** [comment]
Yes, we purposely chose to not send emails for canceled jobs so incomplete results.
Do you still think that it could be useful?

**Comment by tankf33der** [comment]
Let it be that way.
